src/views/submission/dashboard/components/ClinicalChart/index.tsx

```diff
@@ -70,7 +70,6 @@ const useProgramDonorPublishedAnalysisByDateRangeQuery = (
 	const pollingTimeout = useTimeout(30000);
 	const hook = useGatewayQuery(PROGRAM_DONOR_PUBLISHED_ANALYSIS_BY_DATE_RANGE_QUERY, {
 		...options,
-		skip: !dateRangeFrom || !dateRangeTo,
```

Comment:
unclear why we're skipping the query here - no comment - state from above only updates selectively from a useEffect dependency